### DISCUSSION TITLE
社会化抚养 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#430] [General]
# 社会化抚养 - Miao Yu | 于淼

https://yufree.cn/cn/2023/01/04/bring-up-by-professional/

**Comment by earfanfan** [comment]
年初我刚看到你这篇博客的时候，很多内容都觉得不可思议。而现在，我读了费孝通的《生育制度》，又了解过一些东的西的各种东西以后，再想起来跑过来再看一遍，觉得你写得蛮好的。不过，你这好像哪条路都没说死。

话又说回来，我最近写了一篇小说，其中便借了“社会化抚养”的概念。在人伦社会中，人们创建婚姻制度，用各种社会的、法律的因素使得签订婚姻契约的男女要共同达成抚育后代的任务，把一个个体抚育成为可以融入社会的一个分子。基于此，我在我的小说中设定了一种非常极端的情况，就是人工智能背叛了人类，加上极端的环境危机，于是人类社会（主要是统治阶级）决定为了无限提升分工效率，把抚育后代的责任直接从家庭中剥离全由社会完成，婚姻制度就此瓦解，社会快速分化变成了权贵阶级和底层阶级……

但是我最好奇的是，要是将来有一天，三水你有了孩子成为了一个父亲，再看待这些问题的时候又会怎么想

**Reply by yufree** [reply]
最好读下《美丽新世界》，省的写撞车了，反乌托邦题材的作品都会涉及这个问题。
我看这些问题不会因为有没有孩子产生多大变化，我即是旧时代的人，也是新时代的人，但下一代不必受旧时代限制，影响到我为止。

**Reply by earfanfan** [reply]
哇，三水你居然在这个问题上，出乎意料地超级乐观啊。
你说的新时代和旧时代似乎有很明显的分水岭，因而对你的影响会到你为止，而不会影响到你的下一代，或者说不会限制住你这一代的下一代？

我原本好奇如果你有了孩子又会怎么想，脑回路是这样的：三水思考问题的时候倾向于看整体看结构，如果有了孩子，会不会多出一些看待个体的视角。

你肯定也同意社会不会是静止的，但是社会变迁也不会是那么迅速的，快到几年或者十年的时间能让社会割裂出完全没有重叠的新时代的人和旧时代的人。我觉得婚姻制度、生育制度会一直存在，因为一旦婚姻和生育制度瓦解，社会继替会大受影响，社会也需要人类不断生产出新的组成分子，没有的话，就会灭亡。而三水你，本来就是一个社会的分子，一旦你有了孩子，也无可避免地会抚育你的孩子成为新时代新社会的分子，这一点不会变，你跟你的孩子都不能脱离社会而单独存在。所以，我会好奇，到了那个时候，既然社会对你对你的孩子的影响如天罗地网一般无处不在，那么你会不会开始思考个体如何去适应时代变迁、社会变迁，而不仅仅停留在社会结构会随着人类的需求而改变的宏观层面上。

**Reply by earfanfan** [reply]
《美丽新世界》你给我安利过，好像是有一回我给你写信吐槽《1984》的时候，那个时候我就想看这本，但是一直让它在排队。我一直有个困扰是想看的书、在排队的书太多了，安排看的顺序总让我无比纠结……

你倒是提醒我可以搜搜其他反乌托邦题材的作品，看会不会撞车。我想写的是地球启动了一种特殊机制，整个世界会源源不断地冒出一种白烟，这会侵蚀人的记忆，这种情况下，强人工智能得到的最高指令是毁灭人类，于是它们先被人类毁灭了，世界发展到了一种高科技被政府垄断、底层人民又生活得很原始的阶段。我还借了一个你的博客里看到的点，你曾写过，将来世界会发展成一些超大城市群，我的小说世界里也会是这样，因为只有超大城市群有能力汇集更多力量建立出一种风幕系统，能够隔绝白烟。在这个极端设定下，政府不能用大量机器人建造风幕，所以试行社会化抚养政策，发动民众来建立风幕……对了，“三水”是我的小说里面的一个大反派，他建了一座三水城，哈哈哈。

**Reply by yufree** [reply]
不是乐观，只是懒得管下一代怎么想，爱怎么想怎么想，不违法就不反对，跟老子不一样就不支持，最差也就是中立不管。我从来就没感觉啥整体个体视角有啥区别，别扣帽子。时代这东西是个既宏大又虚无的概念，讨论谁适应谁属于虚空讨论，没多大意义。写作这东西如果自娱自乐不用管别人看法，但设定里搞正派反派确实还停留在青少年读物层面。超大城市群也不是我原创，很多作品都有类似设定，而且大概率也会是近未来发展的趋势，不难想到，另外你这设定跟《基地》系列也部分撞了。

**Reply by earfanfan** [reply]
嗯，你一直觉得整体个体视角没撒区别的话，就随你心意继续觉得没区别吧。我陈述我的观点本来就不是想要改变你什么。不过我确实一直觉得这是两个视角，只不过最近读了书渐渐又觉得并不是两个对立的视角了。你打开评论，大概就像是打开门做生意，也不对，但是类似吧，就是打开门之类的。

超大城市群不是你原创就不是呗，反正我有好多新词都是从你这看到的，懒得非得深究来源了，我借的东西本来也不用还。

至于撞车嘛，也正常。不过照我估计你是不会去看我写的小说的，百分百会看不下去，至于其他人嘛，应该不会特地跑过去审判我说我抄袭吧。

在我的小说里面，哈哈，“三水”的设定确实是明面上的大反派，但是也可以说是活得最久、坚持最久、牺牲最多、付出也最多的人了。至于反派这个词，我要是继续纠结这，还不如下班回去多耕点。
